src/pages/Host/HostStair/HostBed.js

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import variables from '../../../styles/variables';
+import HostResult from '../HostResult';
+
+const HostBed = () => {
+  const [people, setPeople] = useState(0);
+  const [bed, setBed] = useState(0);
+  const [room, setRoom] = useState(0);
+  const [bath, setBath] = useState(0);
+
+  const plusPeople = () => {
+    people <= 11 ? setPeople(prev => prev + 1) : setPeople(people);
+  };
```

Comment:
삼항연산자라 읽기 힘든데 이건 어때요? 아 물론 사람바이 사람 회사 바이 회사라서 꼭 수정해주실 필요는 없습니다만
```suggestion
  const plusPeople = () => {
    if(people > 11) return;  // 11이 넘었을때의 방어(물고기 아님)코드 
    setPeople(prev => prev + 1) 
  };
```